datarequest.py

```diff
@@ -700,6 +701,42 @@ def rule_datarequest_review_period_expiration_check(ctx):
         datarequest_process_expired_review_periods(ctx, [result['COLL_NAME'].split('/')[-1] for result in list(qcoll)])
 
 
+@rule.make(inputs=[0], outputs=[1])
```

Comment:
We can remove the rule decorator, because we don't have to call this from the rule language.